### PR TITLE
Carousel positioning

### PR DESCRIPTION
**Pull request by curtisadams28**
ADO card: ☑️ [AB#8176](https://dev.azure.com/S72/SHIFT72/_workitems/edit/8176)

## Description of work
I was unable to find the cause of this bug so I returned the carousel/nav positioning back to the old version. There are a number benefits in addition to fixing the iPad bug:
- The background is contained within `s72-carousel`.
- The variables are easier to understand.
- We won't have to manually adjust the positioning of content on the homepage when the navigation changes height.

### Changes
- The navigation is now `position: absolute`.
- The carousel is `position: relative`.
- The hax `s72-carousel-container-height` variables have been removed.
- `page-detail-padding-top` and `page-padding-top` have been added back.

### Testing
- All pages will need to be tested to ensure that the page-padding variables have been applied everywhere they are needed.
- Homepage content should be pushed down if the carousel is empty.

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [x] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

**Review by l0ud0gg — APPROVED** [review]
without having a wealth of knowledge with css, it looks good to me. i ran the branch up over a few different sites and they all seemed to be functional / look good across different resolutions etc. Maybe @PeculiarGoat would be good to review / give heads up to.

File: site/styles/_pages.scss [hunk 1]
```diff
@@ -76,6 +98,17 @@
   }
 }
 
+.curated-page,
+.collection-page {
```
Comment (PeculiarGoat) [inline site/styles/_pages.scss:102]: Is this block made redundant by the block on line 72?

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Few things to look at.  Also, can you please test every page on the site before this gets merged!

**Review by stajs — APPROVED** [review]
Looks fine, notwithstanding @PeculiarGoat's comments.

For the record, the switch back to absolute positioning and page padding variables is a temporary thing until we sort out proper relative positioning, yeah?

File: site/styles/_meta-detail.scss [hunk 2]
```diff
@@ -120,14 +120,18 @@
   @extend .d-flex;
   flex-direction: column;
   gap: 0 20px;
```
[the remark this answers is not in the document]
Comment (curtisadams28) [reply site/styles/_meta-detail.scss:122]: Changed it to a margin

File: site/styles/_meta-detail.scss [hunk 3]
```diff
@@ -397,9 +401,18 @@ s72-element-switcher,
...
+      padding-left: 20px;
+      padding-right: 20px;
+      padding-top: var(--page-detail-padding-top-md);
+    }
 
     @include media-breakpoint-up(lg) {
-      padding: 0 50px;
+      padding-left: 50px;
+      padding-right: 50px;
+      padding-top: var(--page-detail-padding-top-lg);
```
[the remark this answers is not in the document]
Comment (curtisadams28) [reply site/styles/_meta-detail.scss:415]: Yup. Removed.

File: site/styles/_pages.scss (hunk 1 above)
Comment (curtisadams28) [reply site/styles/_pages.scss:102]: Yes. Removed.

**Review by PeculiarGoat — APPROVED** [review]

**Pull request merged**